src/main/java/DukeExceptions.java

```diff
@@ -0,0 +1,2 @@
+public abstract class DukeExceptions {
```

Comment:
Hmmm... just a `DukeExceptions` may not mean anything much. If you are intending to write custom exceptions, consider extending the `Exceptions` class.